maintenance_evolution.es.Rmd

```diff
@@ -207,6 +207,21 @@ No es necesario que cambies los test de las funciones obsoletas hasta que caduqu
 
 Una vez que se caduca una función, sus tests se eliminan sin más.
 
+## Renombrar paquetes {#renaming-packages}
+
+Renombrar un paquete que está en desarrollo temprano está bien.
+Puede ser la oportunidad, antes de la revisión, de cumplir con nuestros [consejos de nomenclatura](#naming-your-package).
+
+Renombrar un paquete que ya está ampliamente adoptado y/o publicado en CRAN es problemático.
+CRAN tiene una [política](https://cran.r-project.org/web/packages/policies.html) que establece que los nombres de paquetes en CRAN son persistentes y en general no está permitido cambiar el nombre de un paquete.
+El paquete con su antiguo nombre puede ser una dependencia de paquetes, scripts, y aparecer en documentación y entradas de blog, etc.
+
+Cuando se cambia radicalmente la interfaz, es mejor empezar un nuevo paquete desde cero, como [httr2 que es la segunda generación de httr](https://www.tidyverse.org/blog/2023/11/httr2-1-0-0/) ; o crear ediciones de un paquete como para [testthat](https://testthat.r-lib.org/articles/third-edition.html). 
+Si también mantiene el paquete antiguo, puede hacer una depreciación suave del mismo con un mensaje de inicio, como en httr.
+Esto permite a los usuarios y a los autores de paquetes elegir cuándo y si actualizar su código al nuevo paquete o edición.
+Si copias código de otro paquete, asegúrate de reconocer a los autores del código que reutilizas listándolos en DESCRIPTION con un comentario que indique que fueron autores del paquete original.
```

Comment:
```suggestion
Si copias código de otro paquete, asegúrate de reconocer a las personas autoras del código que reutilizas listándolos en DESCRIPTION con un comentario que indique que fueron autores del paquete original.
```

Reply:
We can link to an example here. I think Agroclimatico used to do this with a function from a package that was archived and not maintained @paocorrales @eliocamp is this still true?